src/applications/disability-benefits/all-claims/content/unemployabilityDates.jsx

```diff
@@ -29,18 +28,18 @@ export const dateFieldsDescription = (
     >
       <h5>Date you became too disabled to work</h5>
       <p>
-        This is the date you could no longer work full-time or part time due to
+        This is the date you could no longer work full time or part time due to
         your service-connected disability.
       </p>
       <h5>Date you last worked full-time</h5>
       <p>
-        This is the date you could no longer work full-time due to your
+        This is the date you could no longer work full time due to your
         service-connected disability.
       </p>
       <h5>Date your disability began to affect your full-time employment</h5>
```

Comment:
@peggygannon Should this `full-time` also be `full time`?

Reply:
@peggygannon Let me know if these should be consistent as Chris mentioned; if so I'll make the change and we can merge to staging for visual review